WMF Framework/Theme.swift

```diff
@@ -237,7 +237,7 @@ public class Theme: NSObject {
     }
 
     @objc public lazy var navigationBarBackgroundImage: UIImage = {
-        return UIImage.wmf_image(from: colors.paperBackground)
+        return #imageLiteral(resourceName: "transparent-pixel")
```

Comment:
was this intentional or did I add this when we were considering going to the blur background?

Reply:
I think this happened when we were considering going to the blur bg